.github/workflows/ci-cd.yml

```
@@ -122,11 +125,12 @@ jobs:
             echo "SERVICE=stag" >> $GITHUB_ENV
           fi
 
-      - name: Login to Docker Hub
+      - name: Login to GHCR
         uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+        with:https://github.com/Team-jeong-ho-kim/Daedongyeojido_BE_v4.0
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)